src/components/flat-table/flat-table-row/flat-table-row.style.js

```diff
@@ -259,6 +263,23 @@ const StyledFlatTableRow = styled.tr`
         }
       `}
 
+      ${isDragging &&
+      css`
+        border: ${
+          isInSidebar
+            ? theme.palette.slateTint(40)
+            : theme.palette.slateTint(55)
+        } 2px solid
```

Comment:
```suggestion
        } 2px solid;
```

Needed for when we do upgrade styled-components because v6 doesn't add them in